test/test_cmd.rb

```diff
@@ -0,0 +1,9 @@
+require 'test/unit'
+require_relative '../lib/pmdtester/cmd'
+class TestCmd < Test::Unit::TestCase
+  def test_get_stdout
+    stdout, stderr = PmdTester::Cmd.execute("echo Hello, World!")
+    assert_equal("Hello, World!", stdout);
+    assert_nil(stderr)
+  end
+end
```

Comment:
If you want add test cases to check the exit code of the command, you can use e.g. `/bin/false` to have a failing command....